kbs/src/attestation/intel_trust_authority/mod.rs

```diff
@@ -3,18 +3,27 @@
 // SPDX-License-Identifier: Apache-2.0
 
 use super::Attest;
+use crate::attestation::{generic_generate_challenge, make_nonce};
 use crate::token::{
     jwk::JwkAttestationTokenVerifier, AttestationTokenVerifier, AttestationTokenVerifierConfig,
     AttestationTokenVerifierType,
 };
 use anyhow::*;
 use async_trait::async_trait;
+use attestation_service::HashAlgorithm;
```

Comment:
I had somehow missed this dependency earlier. This is slightly problematic as it is right now. This makes one AS to depend on another AS code just for the sake of `HashAlgorithm` enum. Not only that but `attestation-service/default` makes the build to include all of the CoCo-AS verifier dependencies, such as the vtpm stuff that now makes se CI docker build to fail because the ITA `Dockerfile` does not have `libtss2-dev` installed.

Reply:
Branch updated. I've made a local definition of `HashAlgorithm` to resolve that issue.

Once this and https://github.com/confidential-containers/guest-components/pull/712 land, I plan to tal at consolidating all the definitions though :smile: